modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -4,6 +4,7 @@
 :webserver-user: www-data
 :webserver-group: www-data
 :install-directory: /var/www/owncloud
+:oc-version: 10.4.0
```

Comment:
There are a number of predefined constants (or site-wide attributes) defined at the bottom of `site.yml`. These include `current-version`, which always tracks the latest stable ownCloud version (but only includes the major and minor version). Can you move `oc-version` to site.yml so that it's easier to keep track of, pls?